src/main/webapp/query-control.js

```diff
@@ -216,6 +224,75 @@ async function createStateDropdownList() {
   });
 }
 
+// Set up an autocomplete dropdown, attached to the main location input
+// and states dropdown, to suggest places to users as they type
+function setupAutocompleteLocation() {
+  const autocompleteInput = document.getElementById('location-list');
+  const autocompleteResults = document.getElementById('location');
+  const service = new google.maps.places.AutocompleteService();
+  const defaultLocationOptions = autocompleteResults.innerHTML; // all states
+
+  function displaySuggestions(predictions, status) {
+    if (status != google.maps.places.PlacesServiceStatus.OK) {
```

Comment:
Not sure how the status value works but could use a !== if possible

Reply:
Oh yeah probably, thanks